programs/token/src/instructions/initialize_multisig.rs

```diff
@@ -0,0 +1,103 @@
+use core::{mem::MaybeUninit, slice};
+
+use pinocchio::{
+    account_info::AccountInfo,
+    cpi::invoke_with_bounds,
+    instruction::{AccountMeta, Instruction},
+    program_error::ProgramError,
+    ProgramResult,
+};
+
+/// Maximum number of multisignature signers.
+pub const MAX_MULTISIG_SIGNERS: usize = 11;
+
+/// Initialize a new Multisig.
+///
+/// ### Accounts:
+///   0. `[writable]` The multisig account to initialize.
+///   1. `[]` Rent sysvar
+///   2. ..`2+N`. `[]` The N signer accounts, where N is between 1 and 11.
+pub struct InitializeMultisig<'a, 'b>
+where
+    'a: 'b,
+{
+    /// Multisig Account.
+    pub multisig: &'a AccountInfo,
+    /// Rent sysvar Account.
+    pub rent_sysvar: &'a AccountInfo,
+    /// Signer Accounts
+    pub multisig_signers: &'b [&'a AccountInfo],
```

Comment:
On the token program, we use `signers`.
```suggestion
    pub signers: &'b [&'a AccountInfo],
```